utils/pipeline.py

```diff
@@ -4,11 +4,54 @@
 from typing import Union, List, Dict
 
 
-class IngestPipeline(IngestPipeline):
+class IngestPipelineUtils(IngestPipeline):
+    def run(self, filepath: Union[str, List[str]]) -> xr.Dataset:
```

Comment:
@kefeimo can you remove the run method here? It should be inherited from the parent class. The point of this pipeline is just to add the ability to run the plots separately.

Reply:
Can do. Basically, the `def run` in `utils/pipeline.py` and `tsdat/pipeline/ingest_pipeline.py` are identical, and there is not much override in the template.


<img width="772" alt="image" src="https://user-images.githubusercontent.com/28743873/163660047-ccc8fa91-70b0-4c08-ba8b-c3458b61a9ad.png">